github/exemptions-github.yaml

```diff
@@ -0,0 +1,10 @@
+exemptions:
```

Comment:
I believe this should be moved to `foxsec-automation`.

Reply:
Agreed, but then we're saying this can't land until I finish the plumbing in foxsec-automation. A bit of chicken and egg. An option would be to make an issue to move this. ATM, it just has a "test exemption" -- it does have to be moved before any real data is entered.

Another discussion point -- the modern solution is a mono-repo, but we can't do that. :)